src/js/components/RadioButtonGroup/__tests__/RadioButtonGroup-test.js

```diff
@@ -40,7 +40,13 @@ describe('RadioButtonGroup', () => {
   test('boolean options', () => {
     const component = renderer.create(
       <Grommet>
-        <RadioButtonGroup name="test" options={[true, false]} value />
+        <RadioButtonGroup
+          name="test"
+          options={[
+            { disabled: true, value: '1' },
+            { disabled: false, value: '2' },
+          ]}
```

Comment:
This is changing the test. we still need to have a test an array of boolean values.

Reply:
Based on the RadioButtonGroup documentation https://v2.grommet.io/radiobuttongroup I'm not sure that the options props should be able to take in an array of boolean values.

Reply:
I agree, but I think the intent of this test was to test a `label` that is "true" or "false".
As mentioned in this last week PR https://github.com/grommet/grommet/pull/4180.
@ericsoderberghp feel free to correct me if I'm wrong or clarify what was the test intention.

Reply:
`options={[true, false]}` should be accepted. grommet supports it. It sounds like we need to update the documentation site to reflect that.

Reply:
For those options, I would expect to see labels "1" and "2".

Reply:
I fixed the issues raised from this thread on https://github.com/grommet/grommet/pull/4221 and https://github.com/grommet/grommet-site/pull/174.
@jcfilben please update the snapshots on this PR with its original test after PR #4221 will be merged. It should solve your issue I believe.